Makefile

```diff
@@ -52,7 +53,7 @@ build:
 
 install:
 	# GOPATH/bin must be on your PATH to access these binaries:
-	GO15VENDOREXPERIMENT=1 go install ./src/server/cmd/pachctl ./src/server/cmd/pachctl-doc
+	GO15VENDOREXPERIMENT=1 go install ./src/server/cmd/pachctl ./src/server/cmd/deploy ./src/server/cmd/pachctl-doc
```

Comment:
Good call, how about `pach-deploy`? `pachctl-deploy` suggests to me that it's deploying pachctl.